tests/Unit/Parallel/Test_Main.cpp

```diff
@@ -14,19 +14,24 @@
 #include "Parallel/Main.hpp"
 #include "Utilities/TMPL.hpp"
 #include "tests/Unit/Parallel/ParallelTestChares.hpp"
+#include "tests/Unit/Parallel/ParallelTestClasses.hpp"
 #include "tests/Unit/Parallel/ParallelTestTentacles.hpp"
 #include "tests/Unit/TestHelpers.hpp"
 
 #include "tests/Unit/Parallel/Test_Main.decl.h"
 
-void setup_error_handling() {
+ void setup_error_handling() {
```

Comment:
Why the space here?

Reply:
Don't bother changing the PR, this test is going to change with Algorithm anyway.